src/main/java/org/zalando/nakadi/service/EventPublisher.java

```diff
@@ -157,9 +156,9 @@ private void enrich(final List<BatchItem> batch, final EventType eventType)
         }
     }
 
-    private void traceEventPublishingError(final Scope scope, final BatchItem batchItem, final String error) {
+    private void traceEventPublishingError(final Span span, final BatchItem batchItem, final String error) {
```

Comment:
please, remove the method and put logic in the place where you use it (only one place) and do not use `log` for that bu rather tag event id

Reply:
also use category to identify if event type has metadata `eventType.getCategory()`